dart/test/sentry_client_test.dart

```diff
@@ -1033,6 +1033,42 @@ void main() {
     });
   });
 
+  group('SentryClient ignored errors', () {
+    late Fixture fixture;
+
+    setUp(() {
+      fixture = Fixture();
+      fixture.options.ignoreErrors = ["my-error", "error-.*"];
+    });
+
```

Comment:
let's also add a case where it's partially matching 

e.g the message is "partially matching my-error" and it should ignore it because it matched my-error